examples/index.html

```diff
@@ -55,17 +56,17 @@ <h2 id="examples_by_role_label">Examples by Role</h2>
             <td><code>button</code></td>
             <td>
               <ul>
-                <li><a href="button/button_idl.html">Button  (IDL Version)</a></li>
-                <li><a href="button/button.html">Button</a></li>
+                <li><a href="button/button_idl.html" title>Button  (IDL Version)</a></li>
+                <li><a href="button/button.html" title>Button</a></li>
               </ul>
             </td>
           </tr>
           <tr>
             <td><code>checkbox</code></td>
             <td>
               <ul>
-                <li><a href="checkbox/checkbox-1/checkbox-1.html">Checkbox  (Two State)</a></li>
-                <li><a href="checkbox/checkbox-2/checkbox-2.html">Checkbox  (Mixed-State)</a></li>
+                <li><a href="checkbox/checkbox-1/checkbox-1.html" title="High Contrast Support">Checkbox  (Two State)</a> (<abbr title="High Contrast Support">HC</abbr>)</li>
```

Comment:
I'm not a big fan of having a tooltip that says "High Contrast Support" on the link. It's a rather weird experience for a sighted user, tooltips have a bunch of known accessibility and usability issues, and we already have one on the abbreviation.

If we want a programmatic association on the link itself for screen reader users, we could use `aria-describedby` pointing at a single node.